test_unstructured_ingest/test-ingest-github.sh

```diff
@@ -2,61 +2,35 @@
 
 set -e
 
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 cd "$SCRIPT_DIR"/.. || exit 1
+OUTPUT_FOLDER_NAME=github
+OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
+DOWNLOAD_DIR=$SCRIPT_DIR/download/$OUTPUT_FOLDER_NAME
 
 GH_READ_ONLY_ACCESS_TOKEN=${GH_READ_ONLY_ACCESS_TOKEN:-none}
 
 ACCESS_TOKEN_FLAGS=""
 # to update test fixtures, "export OVERWRITE_FIXTURES=true" and rerun this script
 if [[ "$GH_READ_ONLY_ACCESS_TOKEN" != "none" ]]; then
-   ACCESS_TOKEN_FLAGS="--git-access-token $GH_READ_ONLY_ACCESS_TOKEN"
+    ACCESS_TOKEN_FLAGS="--git-access-token $GH_READ_ONLY_ACCESS_TOKEN"
 elif [[ "$CI" == "true" ]]; then
     echo "Warning: GH_READ_ONLY_ACCESS_TOKEN is not defined in the CI environment."
     echo "This can lead to intermittent failures in test-ingest-github.sh, as non-auth'ed"
     echo "requests are severely rate limited by GitHub."
     echo
-    if [ "$(( RANDOM % 10))" -lt 1 ] ; then
-        # NOTE(crag): proper fix is being tracked here: https://github.com/Unstructured-IO/unstructured/issues/306
-        echo "Skipping ingest 90% of github ingest tests to avoid rate limiting issue."
-        exit 0
-    fi
```

Comment:
we resolved the issue by adding the needed key, so this _shouldn't_ ever be missing in CI. tempted to just fail here now (would rather know if that key was missing in CI?